common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -181,9 +189,19 @@ object PageElement {
       }).toList
 
       case Embed => extractEmbed(element).toList
-
-      case Contentatom => element.contentAtomTypeData.map(d => ContentAtomBlockElement(d.atomId)).toList
-
+      case Contentatom => (extractAtom match {
+        case Some(mediaAtom: MediaAtom) =>
+          mediaAtom.activeAssets.headOption.map(asset => {
+            YoutubeBlockElement(
+              mediaAtom.id, //CAPI ID
+              asset.id,
```

Comment:
For comment consistency:

```suggestion
              asset.id, //YouTube ID
```